substrate-node/pallets/pallet-tft-price/src/lib.rs

```diff
@@ -200,17 +255,21 @@ impl<T: Config> Module<T> {
             http::Error::Unknown
         })?;
 
-        let price = match Self::parse_price(body_str) {
+        let price = match Self::parse_lowest_price_from_request(body_str) {
             Some(price) => Ok(price),
             None => {
                 log::warn!("Unable to extract price from the response: {:?}", body_str);
                 Err(http::Error::Unknown)
             }
         }?;
 
-        log::warn!("Got price: {} cents", price);
+        // Get price for 1 TFT in mUSD
+        let tft_usd = (U64F64::from_num(price) / U64F64::from_num(DST_AMOUNT))
+            .round()
+            .to_num::<u32>();
+        log::warn!("Got price: {} mUSD", tft_usd);
```

Comment:
Why is this a warning? I would think it is information. Maybe even debug info.

Reply:
changed to info